website_sale_stock_control/models/product.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    inventory_availability = fields.Selection([
```

Comment:
Please prefix it with `website_` for indicating their purpose.

Reply:
I called it by similarity as v11, I am thinking in upgrade mode

Reply:
OK, then please put a comment before
```python
# Called the same way as v11 website_sale_stock_control module
```
and make sure also that the values provided are the same.